databricks-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/__main__.py

```diff
@@ -0,0 +1,47 @@
+"""{{ cookiecutter.project_name }} file for ensuring the package is executable
+as `{{ cookiecutter.repo_name }}` and `python -m {{ cookiecutter.python_package }}`
+"""
+import importlib
+from pathlib import Path
+
+from kedro.framework.cli.utils import KedroCliError, load_entry_points
+from kedro.framework.project import configure_project
+
+
+def _find_run_command(package_name):
+    try:
+        project_cli = importlib.import_module(f"{package_name}.cli")
+        # fail gracefully if cli.py does not exist
+    except ModuleNotFoundError as exc:
+        if f"{package_name}.cli" not in str(exc):
+            raise
+        plugins = load_entry_points("project")
+        run = _find_run_command_in_plugins(plugins) if plugins else None
+        if run:
+            # use run command from installed plugin if it exists
+            return run
+        # use run command from `kedro.framework.cli.project`
+        from kedro.framework.cli.project import run
+
+        return run
+    # fail badly if cli.py exists, but has no `cli` in it
+    if not hasattr(project_cli, "cli"):
+        raise KedroCliError(f"Cannot load commands from {package_name}.cli")
+    return project_cli.run
+
+
+def _find_run_command_in_plugins(plugins):
+    for group in plugins:
+        if "run" in group.commands:
+            return group.commands["run"]
+
+
+def main(*args, **kwargs):
+    package_name = Path(__file__).parent.name
+    configure_project(package_name)
+    run = _find_run_command(package_name)
+    run(*args, **kwargs)
```

Comment:
Should we remove it? Given you have create a new entrypoint and this doesn't work for Databricks? Or should we just replace this with the content of `databricks_run.py`

Reply:
Hmm, good question. I would rather `databricks_run.py` as an independent script, to make it clear to users that they need this in their own projects.

Since `__main__.py` is not used, I'll delete it.